dotnetv3/cross-service/AuroraItemTracker/Readme.md

```diff
@@ -0,0 +1,180 @@
+# Track work items in an Aurora Serverless database with the SDK for .NET (v3)
+
+## Overview
+
+This example shows you how to use the AWS SDK for .NET (v3) to create a REST service that lets you do the following:
+
+- Read, write, and update work items that are stored in an Amazon Aurora Serverless database.
+- Use Amazon Simple Email Service (Amazon SES) to send email reports of work items.
+
+## ⚠️ Important
+
+- Running this code might result in charges to your AWS account.
+- Running the tests might result in charges to your AWS account.
+- We recommend that you grant your code least privilege. At most, grant only the minimum permissions required to perform the task. For more information, see [Grant least privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege).
+- This code is not tested in every AWS Region. For more information, see [AWS Regional Services](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services).
+
+## Prerequisites
+
+To run the code in this example, you need the following:
+
++ An AWS account.
++ .NET 6
++ To set up your development environment,
+see [Setting up your AWS SDK for .NET environment](https://docs.aws.amazon.com/sdk-for-net/v3/developer-guide/net-dg-setup.html). 
+
+## Create the resources
+
+Using the AWS AWS Cloud Development Kit (AWS CDK), you can set up the resources required for this example. For more information, see [CDK instructions](https://github.com/awsdocs/aws-doc-sdk-examples/tree/main/resources/cdk/aurora_serverless_app/README.md).
+
+### Create the database table
+
+After you have created the Aurora DB cluster and database, create a table to contain work items. You can do this by using either the AWS Command Line Interface (AWS CLI) or the AWS Management Console.
+AWS CLI
+
+Use the AWS CLI to create the work_items table by running the following command at a command prompt. Before you run, replace the following values with the output from the CloudFormation setup script:
+
+- **CLUSTER_ARN** — Replace with the ARN of the Aurora DB cluster, such as `arn:aws:rds:us-west-2:123456789012:cluster:doc-example-aurora-app-docexampleauroraappcluster-15xfvaEXAMPLE`.
+- **SECRET_ARN** — Replace with the ARN of the secret that contains your database credentials, such as `arn:aws:secretsmanager:us-west-2:123456789012:secret:docexampleauroraappsecret8B-xI1R8EXAMPLE-hfDaaj`.
+- **DATABASE** — Replace with the name of the database, such as `auroraappdb`.
+
+Tip: The caret ^ is the line continuation character for a Windows command prompt. If you run this command on another platform, replace the caret with the line continuation character for that platform.
+
+```
+aws rds-data execute-statement ^
+    --resource-arn "CLUSTER_ARN" ^
+    --database "DATABASE" ^
+    --secret-arn "SECRET_ARN" ^
+    --sql "CREATE TABLE items (id VARCHAR(45), description VARCHAR(400), guide VARCHAR(45), status VARCHAR(400), name VARCHAR(45), archived BOOLEAN);"
```

Comment:
Do these column names match the ones used by your code? I'm not 100% sure (my .NET is rusty), so double check?

Reply:
Double-checked, and yes, they match.